File: server/src/main/kotlin/com/openmpy/server/member/dto/response/MemberGetLocationResponse.kt
package com.openmpy.server.member.dto.response

import com.openmpy.server.member.domain.type.Gender
import java.time.LocalDateTime

data class MemberGetLocationResponse(

    val memberId: Long,
    val thumbnail: String?,
    val nickname: String,
    val bio: String?,
    val gender: Gender,
    val age: Int,
    val likes: Long,
    val distance: Double?,
    val updatedAt: LocalDateTime,
)

File: app/app/Model/Member/Response/MemberGetLocationResponse.swift
struct MemberGetLocationResponse: Codable, Identifiable {

    let memberId: Int64
    let thumbnail: String?
    let nickname: String
    let bio: String?
    let gender: String
    let age: Int
    let likes: Int64
    let distance: Double?
    let updatedAt: String

    var id: Int64 { memberId }
}
